spec/requests/search_results_controller_spec.rb

```diff
@@ -79,6 +79,28 @@
       end
     end
 
+    context 'given a valid params with url_equals filter' do
+      it 'returns a successful response with JSON' do
+        user = Fabricate(:user)
+        Fabricate(:search_result, user_id: user.id, non_adwords_urls: ['wwww.nimblehq.co'])
+        Fabricate(:search_result, user_id: user.id, adwords_top_urls: ['wwww.nimblehq.co'])
+        Fabricate(:search_result, user_id: user.id, non_adwords_urls: ['wwww.nimblehq.co/compass'])
+        Fabricate(:search_result, user_id: user.id, adwords_top_urls: ['wwww.nimblehq.co/compass'])
```

Comment:
After reviewing https://github.com/nimblehq/ic-rails-huey-toby/pull/53/files#diff-cbee06c3149f3c01bef664c25f0d4e6077787d3a0eff949d8f249d4dc39fc9cc, with hinsight, it becomes hard to read if we don't add variables names or some comment for this fabricated data. It would be best to do the following:

```ruby
# Expected matches
Fabricate(:search_result, user_id: user.id, non_adwords_urls: ['www.nimblehq.co'])
Fabricate(:search_result, user_id: user.id, adwords_top_urls: ['www.nimblehq.co'])

# Non-matches
Fabricate(:search_result, user_id: user.id, non_adwords_urls: ['www.nimblehq.co/compass'])
Fabricate(:search_result, user_id: user.id, adwords_top_urls: ['www.nimblehq.co/compass'])
```